src/DataLib/fendl32B_retrofit/tendl_preprocessing.py

```diff
@@ -0,0 +1,27 @@
+# Import packages
+import ENDFtk
+
+def extract_endf_specs(path):
+    """
+    Extract the material ID and MT numbers from an ENDF file.
+
+    Arguments:
+        path (str): File path to the selected ENDF file.
+    
+    Returns:
+        endf_specs (list): List containing the following values from the file:
+            matb (int): Unique material ID extracted from the file.
+            MTs (list): List of reaction types (MT's) present in the file.
+    """
+
+    tape = ENDFtk.tree.Tape.from_file(path)
+    matb = tape.material_numbers[0]
+    # Set MF for cross sections
+    xs_MF = 3
+    file = tape.material(matb).file(xs_MF)
+    # Extract the MT numbers that are present in the file
+    MTs = [MT.MT for MT in file.sections.to_list()]
+
+    endf_specs = [matb, MTs]
+    
+    return endf_specs
```

Comment:
I'm not sure if it's important to formally create a list of this data or if you could just do the following:
```suggestion
        return matb, MTs
```

Reply:
It's not super important, but these decisions sometimes imply specific design needs/choices, and wondering if that's true here.

Reply:
There's not any specific reason why I create the list beforehand, I can remove it to simplify things.